packages/core/src/EntityManager.ts

```diff
@@ -163,7 +164,8 @@ export class EntityManager<D extends IDatabaseDriver = IDatabaseDriver> {
   async find<
     Entity extends object,
     Hint extends string = never,
-  >(entityName: EntityName<Entity>, where: FilterQuery<Entity>, options: FindOptions<Entity, Hint> = {}): Promise<Loaded<Entity, Hint>[]> {
+  >(entityName: EntityName<Entity>, where: FilterQuery<Entity>, _options: FindOptions<Entity, Hint> = {}): Promise<Loaded<Entity, Hint>[]> {
```

Comment:
please dont rename the parameters, there is no need for this, you just alter the options interface but retain the type, so you can just reassign to the same `options` variable
```suggestion
  >(entityName: EntityName<Entity>, where: FilterQuery<Entity>, options: FindOptions<Entity, Hint> = {}): Promise<Loaded<Entity, Hint>[]> {
```

Reply:
Resolved